api/types.proto

```diff
@@ -117,6 +117,8 @@ message NodeStatus {
 
 	State state = 1;
 	string message = 2;
+	// Address is the node's IP address as observed by the manager
+	string address = 3;
```

Comment:
We commonly abbreviate this as `addr` (see types.proto, raft.proto and objects.proto).